File: Extras/025 - Find the Median/FindTheMedian.rb
def findMedian(arr)
  sorted_arr = arr.sort()

  len = sorted_arr.length

  if len % 2 > 0
    sorted_arr[(len - 1) / 2]
  else
    (sorted_arr[len / 2] + sorted_arr[(len / 2) - 1]) / 2
  end
end

p findMedian([2, 6, 4, 5, 7])

File: Extras/025 - Find the Median/FindTheMedian.js
function findMedian(arr) {
  arr.sort((a, b) => b - a);

  let len = arr.length;

  if (len % 2 > 0) {
    return arr[(len - 1) / 2];
  } else {
    return (arr[len / 2] + arr[(len / 2) - 1]) / 2;
  }
}

console.log(findMedian([2, 6, 4, 5, 7]));
